src/ui/hooks/query/useSettingsLastActiveVaultId.ts

```diff
@@ -1,15 +1,22 @@
 import { useQuery } from '@tanstack/react-query';
-import { uiMain } from '@/ui/index';
 import { IpcWorkerMessageType } from '@/lib/ipc/ipcWorker';
+import { useContext } from 'react';
```

Comment:
Move the import to "external libraries" "block", on line 2 (and add return line after)

Reply:
Can we have a tool that does this for us? I don't mind this being a rule as long as we have tool that does it automatically.

Otherwise, if we have to do it manually, I oppose. Its just too much work for the value it provides.

Reply:
There's probably a eslint rule for that. I'll try to find it.

Reply:
Maybe this? https://www.npmjs.com/package/eslint-plugin-ordered-grouped-import

Ideally we select one that works with `--fix`